### PR TITLE
Add is_ok and is_error, simple predicates for testing a result.  This…

### PR DESCRIPTION
**Pull request by orbitz**
… is useful for tests or asserts

### CODE REVIEW

File: src/core/CCResult.mli [hunk 1]
```diff
@@ -87,6 +87,12 @@ val fold : ok:('a -> 'b) -> error:('err -> 'b) -> ('a, 'err) t -> 'b
 (** [fold ~ok ~error e] opens [e] and, if [e = Ok x], returns
     [ok x], otherwise [e = Error s] and it returns [error s]. *)
 
+val is_ok : ('a, 'err) t -> bool
+(** Return true if Ok *)
```
Comment (c-cube) [inline src/core/CCResult.mli]: can you add `@since NEXT_RELEASE` to these functions?

**Comment by c-cube** [comment]
Also, you can add yourself to `AUTHORS` ;-)

**Comment by orbitz** [comment]
@c-cube Updated

**Comment by c-cube** [comment]
Thanks!

**Pull request merged**